pkg/connector/userinfo.go

```diff
@@ -10,12 +10,27 @@ import (
 	"maunium.net/go/mautrix/bridgev2"
 	"maunium.net/go/mautrix/bridgev2/networkid"
 
+	"go.mau.fi/mautrix-meta/pkg/messagix/socket"
 	"go.mau.fi/mautrix-meta/pkg/messagix/types"
 	"go.mau.fi/mautrix-meta/pkg/metaid"
 	"go.mau.fi/mautrix-meta/pkg/msgconv"
 )
 
 func (m *MetaClient) GetUserInfo(ctx context.Context, ghost *bridgev2.Ghost) (*bridgev2.UserInfo, error) {
+	if ghost.Name == "" {
+		contactID := metaid.ParseUserID(ghost.ID)
+		resp, err := m.Client.ExecuteTasks(&socket.GetContactsFullTask{
+			ContactID: contactID,
+		})
+		log := m.UserLogin.Log
+		if err != nil {
+			log.Trace().Any("ContactID", contactID).Any("err", err).Msg("GetContactsFullTask failed")
```

Comment:
The log is not necessary since an error is returned and the caller will log it

```suggestion
		if err != nil {
```